Assets/MRTK/Providers/Oculus/XRSDK/MRTK-Quest/Editor/OculusXRSDKHandtrackingConfigurationChecker.cs

```diff
@@ -56,13 +56,67 @@ public static class OculusXRSDKHandtrackingConfigurationChecker
         private static readonly string[] Definitions = { "OCULUSINTEGRATION_PRESENT" };
 
         /// <summary>
-        /// Detects if the Oculus Integration package is present and updates the AsmDefs with the appropriate definitions and references.
+        /// Integrate MRTK and the Oculus Integration Unity Modules if the Oculus Integration Unity Modules is in the project. If is are not in the project, display a pop up window.
         /// </summary>
         [MenuItem("Mixed Reality Toolkit/Utilities/Oculus/Integrate Oculus Integration Unity Modules")]
-        internal static void ConfigureOculusIntegration()
+        internal static void IntegrateOculusWithMRTK()
+        {
+            // Check if Oculus Integration package is present
+            bool OculusIntegrationPresent = DetectOculusIntegrationDefine();
+
+            if (!OculusIntegrationPresent)
+            {
+                EditorUtility.DisplayDialog(
+                    "Oculus Integration Package Not Detected",
+                    "The Oculus Integration Package could not be found in this project, please import the assets into this project. The assets can be found here: " +
+                        "https://assetstore.unity.com/packages/tools/integration/oculus-integration-82022",
+                    "OK");
+            }
+
+            // Update the ScriptingDefinitions depending on the presence of the Oculus Integration Unity Modules
+            ReconcileOculusIntegrationDefine();
```

Comment:
Looks like `ReconcileOculusIntegrationDefine()` also calls `DetectOculusIntegrationDefine()`. Can the bool from the call in this method be passed down instead? Finding all files _could_ be an expensive operation depending on the number of files.